build/tasks/validate.js

```diff
@@ -307,5 +301,170 @@ function validateRule({ tags, metadata }) {
+    criterionRegex: /^EN-9\.[1-4]\.[1-9]\.\d{1,2}$/,
+    wcagLevelRegex: /wcag21?aa?/
+  }
+];
+
+function findTagIssues(tags) {
+  const issues = [];
+  const catTags = tags.filter(tag => tag.startsWith('cat.'));
+  const bestPracticeTags = tags.filter(tag => tag === 'best-practice');
+  const miscTags = tags.filter(tag => uniqueTags.includes(tag));
+
+  // Category
+  if (catTags.length !== 1) {
+    issues.push(`Must have exactly one cat. tag, got ${catTags.length}`);
+  }
+  if (catTags.length && !categories.includes(catTags[0].slice(4))) {
+    issues.push(`Invalid category tag: ${catTags[0]}`);
+  }
+  if (!startsWith(tags, catTags)) {
+    issues.push(`Tag ${catTags[0]} must be before ${tags[0]}`);
+  }
+  tags = removeTags(tags, catTags);
+
+  // Best practice
+  if (bestPracticeTags.length > 1) {
+    issues.push(
+      `Only one best-practice tag is allowed, got ${bestPracticeTags.length}`
+    );
+  }
+  if (!startsWith(tags, bestPracticeTags)) {
+    issues.push(`Tag ${bestPracticeTags[0]} must be before ${tags[0]}`);
+  }
+  tags = removeTags(tags, bestPracticeTags);
+
+  const standards = {};
+  // WCAG, Section 508, Trusted Tester, EN 301 549
+  for (const {
+    name,
+    standardRegex,
+    criterionRegex,
+    wcagLevelRegex
+  } of standardsTags) {
+    const standardTags = tags.filter(tag => tag.match(standardRegex));
+    const criterionTags = tags.filter(tag => tag.match(criterionRegex));
+    if (!standardTags.length && !criterionTags.length) {
+      continue;
+    }
+
+    standards[name] = {
+      name,
+      standardTag: standardTags[0] ?? null,
+      criterionTags
+    };
+    if (bestPracticeTags.length) {
+      issues.push(`${name} tags cannot be used along side best-practice tag`);
+    }
+    if (standardTags === 0) {
+      issues.push(`Expected one ${name} tag, got 0`);
+    } else if (standardTags.length > 1) {
+      issues.push(`Expected one ${name} tag, got: ${standardTags.join(', ')}`);
+    }
+    if (!criterionTags.length) {
+      issues.push(
+        `Expected at least one ${name} criterion tag, got ${criterionTags.length}`
+      );
+    }
+
+    if (wcagLevelRegex) {
+      const wcagLevel = standards.WCAG.standardTag;
+      if (!wcagLevel.match(wcagLevelRegex)) {
+        issues.push(`${name} rules not allowed on ${wcagLevel}`);
+      }
+    }
+
+    // Must have the same criteria listed
+    if (name === 'EN 301 549') {
+      const wcagCriteria = standards.WCAG.criterionTags.map(tag =>
+        tag.slice(4)
+      );
+      const enCriteria = criterionTags.map(tag =>
+        tag.slice(5).replaceAll('.', '')
+      );
+      if (
+        wcagCriteria.length !== enCriteria.length ||
+        !startsWith(wcagCriteria, enCriteria)
+      ) {
+        issues.push(
+          `Expect WCAG and EN criteria numbers to match: ${wcagCriteria.join(
+            ', '
+          )} vs ${enCriteria.join(', ')}}`
+        );
+      }
+    }
+
+    if (!startsWith(tags, bestPracticeTags)) {
```

Comment:
You've already validated against best practice tags up above (line 401) so we shouldn't need to do so again here.